src/Files.App/ViewModels/Properties/Items/FileProperty.cs

```diff
@@ -324,12 +324,15 @@ public async static Task<List<FileProperty>> RetrieveAndInitializePropertiesAsyn
 		private static readonly Dictionary<string, Func<object, string>> DisplayFuncs = new Dictionary<string, Func<object, string>>()
 		{
 			{ "DivideBy1000", input => (((uint) input)/1000).ToString() },
-			{ "FormatDuration", input => new TimeSpan(Convert.ToInt64(input)).ToString("hh':'mm':'ss")},
+			{ "FormatDuration", input => TimeSpanToString(new TimeSpan(Convert.ToInt64(input)))},
 			{ "Fraction" , input => ((double)input).ToFractions(2000)},
 			{ "AddF" , input => $"f/{(double)input}"},
 			{ "AddISO" , input => $"ISO-{(UInt16)input}"},
 			{ "RoundDouble" , input => $"{Math.Round((double)input)}"},
 			{ "UnitMM" , input => $"{(double)input} mm"},
 		};
+
+		private static string TimeSpanToString(TimeSpan t)
```

Comment:
Is it worth creating a helper/converter for this so that it's reusable?

Reply:
Are there any other places that might use it? The format of the property values is consolidated here, so I think we can leave it as it is.

Reply:
Not that I know of